### DISCUSSION TITLE
后台服务端是active，客户端连上了无法使用

### DISCUSSION THREAD

**Discussion by Tsiyuki1** [apernet/hysteria#1012] [Q&A]
![屏幕截图 2024-03-27 163739](https://github.com/apernet/hysteria/assets/141389812/441d7b8b-e94f-4f8b-9f43-ed849e69837e)
![屏幕截图 2024-03-27 163845](https://github.com/apernet/hysteria/assets/141389812/1fd40feb-db1c-40d5-89c3-dfe77dc93b66)
我用的是acme证书，域名也是没有问题的

**Comment by Gol2dragon** [comment]
请检查服务器端是否开启了防火墙，如开启了就需要放行相应端口，也可以将防火墙关闭了

**Comment by meower1** [comment]
In case your struggling to write your own backend, you can use this premade hysteria panel which has the necessary user management functionality https://github.com/jonssonyan/h-ui
